April23/news.md

```diff
@@ -0,0 +1,9 @@
+# News
+
+## FlashFest organised for April Fools’ snapshot
+
+This month of course comes with April Fools' Day, and that means a Minecraft April Fools' update! In this year's offering we saw a feature in which players regularly vote on "proposals", each of which would make a small change to the game, ranging from changing the colour of grass to... turning the game French? C'est bizarre. Anyway, with this snapshot came an impromptu ModFest, dubbed a "FlashFest". The competition gave contestants 24 hours to make anything they could targeting this snapshot. Some of the brilliant and wildly surreal submissions included a "Bottle of Gender" (what), a mod that allows proposals to be converted to items and then shot out of a gun (???) and a mod with a proposal which instils a sudden generosity into villagers, making them offer all trades for free (the most unrealistic one by far). The competition was a great success, and we hope to see more of these FlashFests in the future - if you want to see all the submissions, go and check out ModFest's Discord (https://discord.gg/gn543Ee).
+
+## MCProHosting faces potential layoffs
+
+The “world leader” in server hosting appears to be in trouble, according to a now deleted tweet. The tweet, which namechecks fellow hosting company Nitrado, states that “all” the staff got laid off, although this may be hyperbolic, given that there are still staff members in the service’s Discord. The mention to Nitrado is significant as their parent company acquired MCProHosting in 2021, which could mean that MCProHosting could be closing, or is downsizing significantly. If the tweet is true, it is a worrying development, and also a reminder that server hosting companies can often be unreliable. Make your backups, people!
```

Comment:
`The mention to Nitrado is significant as their parent company acquired MCProHosting in 2021, which could mean that MCProHosting could be closing, or is downsizing significantly.`

This sentence is a bit confusing, had to read it 2 or 3 times. Maybe:

`The mention to Nitrado is significant as their parent company acquired MCProHosting in 2021, which could mean that MCProHosting could be closing or downsizing significantly`.

I feel like that makes it flow better personally.

Reply:
Will do, also replacing `mention to` with `mention of` for grammatical correctness